code/modules/mob/living/carbon/carbon.dm

```diff
@@ -237,8 +237,12 @@
 				H.w_uniform.add_fingerprint(M)
 
 			var/show_ssd
+			var/target_organ_exists = FALSE
 			var/mob/living/carbon/human/H = src
-			if(istype(H)) show_ssd = H.species.show_ssd
+			if(istype(H))
+				show_ssd = H.species.show_ssd
+				var/obj/item/organ/external/O = H.get_organ(M.targeted_organ)
+				target_organ_exists = !(!O || !O.is_usable())
```

Comment:
are you sure you want to do NOT on (NOT OR NOT) statement?
Is it some kind of joke or what? `! ^ !` = nothing, just`(O || O.is_usable())` will confirm that target EO exists and can be used

Reply:
It converts O, which is normally of type `obj/item/organ/external`, into a boolean type. I'm not sure how DM deals with the truthiness of more complex types like this, and frankly I've never tried it either; this is just a pattern I've seen elsewhere in DM code (it's also common in JavaScript and such).

Reply:
Cmon bruh, byond can understand `if(O || O.is_usable())`. BTW maybe you need to write `&&` instead of `||` ? Like it exists and by the way it works fine.
Anyway do what you want, but "written somewhere by somebody and it just works so i copypasted" not the best approach.

Reply:
`&&` would be needed if I didn't use the double-negative.

I guess since in DM logical operators return a boolean type instead of one of the operands like in eg. Javascript, just `if(O && O.is_usable())` should be enough. I'll change it to that.